src/Endpoints/Delegates/HandlesSettings.php

```diff
@@ -222,15 +222,20 @@ public function resetSynonyms(): array
     // Settings - Filterable Attributes
 
     /**
-     * @return list<non-empty-string>
+     * @return list<non-empty-string>|list<array{attributePatterns: list<non-empty-string>, features?: array{facetSearch: bool, filter: array{equality: bool, comparison: bool}}}>
```

Comment:
on what cases api will return two different responses?

Reply:
If the `filterableAttributes` were configured with the old syntax (list of strings), then the API returns a list of strings.

If configured with the new syntax:

```php
$index->updateFilterableAttributes([
    [
        'attributePatterns' => ['title'],
        'features' => [
            'facetSearch' => true,
            'filter' => [
                'equality' => true,
                'comparison' => false,
            ],
        ],
    ],
]);
```

The previous tests not breaking confirm this, unless I got something wrong. I suspect this is not to cause regressions in an existing codebase using the old syntax.